node/tests/GlideClient.test.ts

```diff
@@ -667,7 +677,9 @@ describe("GlideClient", () => {
                 );
 
                 // re-load library with replace
-                expect(await client.functionLoad(code, true)).toEqual(libName);
+                expect(
+                    await client.functionLoad(code, { replace: true }),
```

Comment:
also need a test for functionLoad that takes a decoder: Decoder.bytes that returns Binary.